raiwidgets/raiwidgets/error_analysis_dashboard_input.py

```diff
@@ -37,7 +37,8 @@ def __init__(
             model_task,
             metric,
             max_depth,
-            num_leaves):
+            num_leaves,
+            sample_dataset):
```

Comment:
Should this be None by default? Just wondering because of the logic below

Reply:
this is the dashboard input, none of the parameters are optional, the user-facing API in error_analysis_dashboard.py has the optional parameters